shared/js/ui/templates/site.es6.js

```diff
@@ -41,13 +41,13 @@ module.exports = function () {
       </a>
     </li>
     <li class="site-info__li--toggle padded ${this.model.isWhitelisted ? '' : 'is-active'}">
-      <h2 class="site-info__protection js-site-protection">Site Privacy Protection</h2>
-      <h2 class="is-hidden site-info__protection js-site-protection-whitelisted">
+      <h2 class="is-transparent site-info__protection--is-whitelisted js-site-protection-whitelisted">
```

Comment:
BEM naming nitpick - feels weird for this element to have the `site-info__protection--is-whitelisted` class, but not the `site-info__protection` class. Worth just calling it e.g. `site-info__whitelist-status` or similar?